services/vaadin-directory/vaadin-directory-rating-count.service.js

```diff
@@ -12,7 +12,7 @@ module.exports = class VaadinDirectoryRatingCount extends BaseVaadinDirectorySer
   static get route() {
     return {
       base: 'vaadin-directory',
-      pattern: ':which(rc|rating-count)/:packageName',
+      pattern: ':alias(rc|rating-count)/:packageName',
```

Comment:
These Vaadin Directory aliases should be changed to redirects, though that doesn't need to be handled in this PR.

Reply:
In a case like this, do you think there's a big advantage to defining an extra object which issues a redirect so that we can change the route from `'vaadin-directory/:alias(rc|rating-count)/:packageName'` to `'vaadin-directory/rc/:packageName'`? Given it doesn't make the route excessively complex, I'd tend towards leaving them as they are and avoiding the extra request.

Reply:
We have a lot of API surface and I think it's nice to pare it down where we can, and particularly it seems nice to advertise a single canonical route rather than two options. Though I guess that is already the case in the published examples.